### ISSUE TITLE
Inconsistence issue regarding the ventilation and dialysis duration between chartevents.csv and procedureevents.csv

### ISSUE THREAD

**Issue by yukuntan92** [MIT-LCP/mimic-code#1615]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.mit.edu/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description
Hi,

I am trying to extract the ventilation and dialysis duration information from MIMIC-IV dataset. I saw the sql file which extracted the ventilation duration (by using some logic to determine the duration) and dialysis (no duration) information using chartevents.csv. I noticed that procedureevents.csv contains the start and end time for those, but seems a little bit inconsistent with what extracted from chartevents.csv. I was wondering why not extract from procedureevents.csv directly? Is it because the records from procedureevents.csv are incomplete or inaccurate? Thank you.

**Comment by alistairewj** [comment]
Yes, procedure events is inconsistent, and it's use depends on provider preference. In general I've found the following levels of agreement

- CRRT: very good, though the procedure derived data tends to omit pauses/gaps
- Ventilation: reasonable
- NIV: procedure events misses many events

Of course you can double check this by cross-referencing the data. In general I'd say the procedure events table is sensitive (real events are captured) but misses events. It depends on your study design whether this is problematic.
